book/src/state/queries.md

```diff
@@ -26,7 +26,7 @@ for frobnicator in hobo::find::<&mut Frobnicator>() {
 }
 ```
 
-Queries are tuples of `& T`, `&mut T` or `With<T>` where `T` is some component or, as a special case, `Entity`. The result of `hobo::find` (or `hobo::find_one`) are tuples where each member is what was requested by the query (`With<T>` will always return `true` in its position because any entity that doesn't have `T` won't be included in the output).    
+Queries are tuples of `& T`, `&mut T` or `With<T>` where `T` is some component or, as a special case, `Entity`. The result of `hobo::find` (or `hobo::find_one`) are tuples where each member is what was requested by the query.    
```

Comment:
you should still specify that `With<T>` will insert a value that should be ignored, it might be a surprise/gotcha otherwise